src/components/Calendar/CheckinModal.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { Modal, Button, Switch } from 'antd'
+import { CheckCircleOutlined, CheckCircleTwoTone } from '@ant-design/icons';
+import '../../less/calendar.less'
+
+const CheckinModal = (props) => {
+
+    if(!props.details.start) {
+        return(
+            <>
+                loading
+            </>
+        )
+    }
+
+    console.log(props.details)
+
+    const handleOk = () => {
+        props.setIsModalVisible(false);
+      };
+    
+      const handleCancel = () => {
+        props.setIsModalVisible(false);
+      };
```

Comment:
Hey, I was wondering if these two functions can become one since they both do the same thing. Maybe call it handleModalVisibility or something? And then link it to both 'ok' and 'cancel' buttons?

Reply:
Yea, good point. The only reason I think maybe to keep it in is because the ok and cancel button functionality may change. However, you're right there is no need to have two functions.